### PR TITLE
framework/ble_manager: add check connection function

### PR DESCRIPTION
**Pull request by an4967**
- add check connection function
- add usage in ble_rmc for example

### CODE REVIEW

File: framework/src/ble_manager/ble_manager_state.c [hunk 1]
```diff
@@ -138,6 +138,21 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 
 		ret = ble_drv_delete_bond_all(mode);
 	} break;
+	case BLE_EVT_CMD_CONN_IS_ACTIVE: {
+		BLE_STATE_CHECK;
+
+		blemgr_msg_params *param = (blemgr_msg_params *)msg->param;
+		trble_conn_handle con_handle = *(trble_conn_handle *)param->param[0];
```
Comment (jeongchanKim) [inline framework/src/ble_manager/ble_manager_state.c:145]: Is it okay not to check `param` is NULL or not?

File: os/net/blemgr/bledev.c [hunk 2]
```diff
@@ -124,6 +124,34 @@ int bledev_handle(struct bledev *dev, lwnl_req cmd, void *data, uint32_t data_le
...
 	}
 	break;
+	case LWNL_REQ_BLE_CONN_IS_ACTIVE:
+	{
+		bool *is_active = NULL;
+		trble_conn_handle con_handle = 0;
+
+		lwnl_msg_params param = { 0, };
+		if (data != NULL) {
+			memcpy(&param, data, data_len);
```
Comment (jeongchanKim) [inline os/net/blemgr/bledev.c:134]: For using `memcpy`, please include `string.h`.

File: framework/src/ble_manager/ble_manager_state.c (hunk 1 above)
Comment (an4967) [reply framework/src/ble_manager/ble_manager_state.c:145]: Yes, It is guranteed.  This functions are not for general users.

File: os/net/blemgr/bledev.c (hunk 2 above)
Comment (an4967) [reply os/net/blemgr/bledev.c:133]: Ok, i will add it

File: framework/include/ble_manager/ble_manager.h [hunk 3]
```diff
@@ -90,3 +85,7 @@ ble_result_e ble_manager_deinit(void);
  *
  ****************************************************************************/
 ble_result_e ble_manager_get_mac_addr(uint8_t mac[BLE_BD_ADDR_MAX_LEN]);
+
```
Comment (jeongarmy) [inline framework/include/ble_manager/ble_manager.h:88]: Please include `#include <stdbool.h>` for `bool` type.

Comment (an4967) [reply framework/include/ble_manager/ble_manager.h:90]: OK, I will add it

**Review by pillip8282 — APPROVED** [review]
lgtm

**Pull request merged**